### PR TITLE
Fix(refs: t34474): incorrect welcomepage

### PR DESCRIPTION
**Pull request by elmasdemos**
**Ticket:** https://yaits.demos-deutschland.de/T34474#834922

### Description: 

- the welcome page shall have more common formulation/content

### PR Checklist

- [x] Link all relevant tickets
- [x] Move the tickets on the board accordingly

### CODE REVIEW

**Review by hwiem — APPROVED** [review]
lgtm, code review only

File: templates/bundles/DemosPlanCoreBundle/DemosPlanUser/gateway_newUser.html.twig [hunk 1]
```diff
@@ -1,5 +1,7 @@
 {% extends '@DemosPlanCore/DemosPlanCore/base_oeb.html.twig' %}
+
 {% if templateVars is defined %}
+
```
Comment (muellerdemos) [inline templates/bundles/DemosPlanCoreBundle/DemosPlanUser/gateway_newUser.html.twig:4]: I'd only use a blank line here if this section would be dense codewise.

File: templates/bundles/DemosPlanCoreBundle/DemosPlanUser/gateway_newUser.html.twig [hunk 2]
```diff
@@ -38,26 +39,42 @@
 
                 {% if feedback.success_message is defined %}
                     <p class="flash flash-confirm">
-                        <i class="fa fa-check-circle" aria-hidden="true"></i>
+                        <i
+                            class="fa fa-check-circle"
+                            aria-hidden="true"></i>
```
Comment (muellerdemos) [inline templates/bundles/DemosPlanCoreBundle/DemosPlanUser/gateway_newUser.html.twig:44]:
```suggestion
                        <i
                            class="fa fa-check-circle"
                            aria-hidden="true">
                        </i>
```

Comment (muellerdemos) [reply templates/bundles/DemosPlanCoreBundle/DemosPlanUser/gateway_newUser.html.twig:44]: This way we can see easier where the tag ends - and other occurences too

File: templates/bundles/DemosPlanCoreBundle/DemosPlanUser/gateway_newUser.html.twig [hunk 3]
```diff
@@ -38,26 +39,42 @@
...
+                    </h3>
 
                     <label class="u-mb-0_5">
                         {{ "email"|trans }}
-                        <input type="text" class="layout__item" value="{% if user is defined %}{{- user.email|default("-") -}}{% endif %}" disabled>
+                        <input
+                            type="text"
+                            class="layout__item"
+                            value="{% if user is defined %}{{- user.email|default("-") -}}{% endif %}"
+                            disabled>
```
Comment (muellerdemos) [inline templates/bundles/DemosPlanCoreBundle/DemosPlanUser/gateway_newUser.html.twig:77]: why is this input field always disabled, I wonder if we need this if it is always disabled.

File: templates/bundles/DemosPlanCoreBundle/DemosPlanUser/gateway_newUser.html.twig [hunk 4]
```diff
@@ -71,9 +88,19 @@
...
+                                        rel="noopener noreferrer">
+                                        {{ 'newsletter.subscribe'|trans }}
+                                    </a>
+
                                 {% else %}
-                                    <input type="checkbox" name="newsletter" {{ user.wantsNewsletter is defined and user.wantsNewsletter == true? 'checked="true"' : '' }}>
+                                    <input
+                                        type="checkbox"
+                                        name="newsletter"
+                                        {{ user.wantsNewsletter is defined and user.wantsNewsletter == true? 'checked="true"' : '' }}>
```
Comment (muellerdemos) [inline templates/bundles/DemosPlanCoreBundle/DemosPlanUser/gateway_newUser.html.twig:103]:
```suggestion
                                        {{ user.wantsNewsletter is defined and user.wantsNewsletter == true ? 'checked="true"' : '' }}>
```

**Review by muellerdemos — COMMENTED** [review]
I was too slow ;(

File: templates/bundles/DemosPlanCoreBundle/DemosPlanUser/gateway_newUser.html.twig (hunk 1 above)
Comment (elmasdemos) [reply templates/bundles/DemosPlanCoreBundle/DemosPlanUser/gateway_newUser.html.twig:4]: I learned that our convention is to put a blink line before every if condition

Comment (salisdemos) [reply templates/bundles/DemosPlanCoreBundle/DemosPlanUser/gateway_newUser.html.twig:4]: before, but not after ;-)
```
const some  = code

if(some) {
  someCode()

  return foo
} else {
  return bar
}
```

Comment (hwiem) [reply templates/bundles/DemosPlanCoreBundle/DemosPlanUser/gateway_newUser.html.twig:4]: ...unless the after is also a before:

```
{% if templateVars is defined %}

  {% if templateVars.title is defined %}
	 // something here
  {% endif %}

{% endif %}
```

Comment (salisdemos) [reply templates/bundles/DemosPlanCoreBundle/DemosPlanUser/gateway_newUser.html.twig:4]: true

**Pull request merged**